src/ios/OpenTokPlugin.m

```diff
@@ -464,8 +481,10 @@ - (void)subscriberDidConnectToStream:(OTSubscriberKit*)sub{
     NSMutableDictionary* eventData = [[NSMutableDictionary alloc] init];
     NSString* streamId = sub.stream.streamId;
     [eventData setObject:streamId forKey:@"streamId"];
+    NSString* sessionId = sub.session.sessionId;
+    NSString* sessionEventName = [NSString stringWithFormat:@"sessionEvents:%@", sessionId];
     [self triggerJSEvent: @"subscriberEvents" withType: @"connected" withData: eventData];
-    [self triggerJSEvent: @"sessionEvents" withType: @"subscribedToStream" withData: eventData]; // Backwards compatibility
+    [self triggerJSEvent: sessionEventName withType: @"subscribedToStream" withData: eventData]; // Backwards compatibility
```

Comment:
Backwards compatibility with what ?

Reply:
I don't know, it's not my comment, it was here before I touched it